cmake/Findzstd.cmake

```diff
@@ -4,14 +4,17 @@ endif()
 
 set(zstd_FOUND FALSE)
 
-find_package(PkgConfig)
-if(PKG_CONFIG_FOUND)
-  pkg_search_module(PC_ZSTD libzstd)
-  find_library(ZSTD_LIBRARY zstd HINTS ${PC_ZSTD_LIBDIR} ${PC_ZSTD_LIBRARY_DIRS})
-  find_path(ZSTD_INCLUDE_DIR zstd.h HINTS ${PC_ZSTD_INCLUDEDIR} ${PC_ZSTD_INCLUDE_DIRS})
-else()
-  find_library(ZSTD_LIBRARY zstd)
-  find_path(ZSTD_INCLUDE_DIR zstd.h)
+# Always use internet zstd on apple platforms
```

Comment:
Added a note, because the installed stuff is all x86 but i need to use a locally compiled version since I'm cross compiling for arm targets.

Reply:
OK, but what about people who build ccache on macOS themselves? I don't think that we should force a download of zstd for macOS builds just because CI needs it.

Reply:
Before 2c742c2c7ca9 it was possible to force downloading with `ZSTD_FROM_INTERNET=ON`. I think that it would be a good idea to change `ZSTD_FROM_INTERNET` to be a tristate variable with values ON, OFF or AUTO instead of adding `FORCE_INTERNET_ZSTD`. But I'll make that change that after merging this.

Reply:
Done in fe5ddc9e79c39ba1b98f124561ce1b3f2cec3c19.